### ISSUE TITLE
Training would be not convergence for backbone res34

### ISSUE THREAD

**Issue by gneworld** [cfzd/Ultra-Fast-Lane-Detection#36]
Thanks for your great job. When I train the model with resnet34 backbone from scratch, other config like this,
optimizer = 'Adam'  #['SGD','Adam']
learning_rate = 0.025
weight_decay = 1e-4
momentum = 0.9

scheduler = 'multi' #['multi', 'cos']
steps = [25, 50, 75, 100]
gamma  = 0.1
warmup = 'linear'
warmup_iters = 695

training seems that wouldn't be convergence, after 50 epochs, looking forward to your reply

**Comment by cfzd** [comment]
@gneworld 
Could you please provide the evaluation results or the training curve in tensorboard?

**Comment by gneworld** [comment]
![ggg](https://user-images.githubusercontent.com/30924021/87265653-ebc7e880-c4f5-11ea-9e44-4cfa51cae12f.jpg)

@cfzd  that is my  training curve in tensorboard

**Comment by cfzd** [comment]
@gneworld 
It seems that it didn't converge.  You can change the learning rate to `1e-4`.

According to my experience, the learning rate for adam optimizer is much smaller than the one of SGD. `0.025` is too big for adam optimizer.

**Comment by gneworld** [comment]
thanks much
